pkg/devicemapper/devmapper.go

```diff
@@ -309,7 +309,7 @@ func FindLoopDeviceFor(file *os.File) *os.File {
 	return nil
 }
 
-// UdevWait wakes any processes that are waiting for udev to complete the specified cookie. (?)
+// UdevWait wakes any processes that are waiting for udev to complete the specified cookie.
```

Comment:
@vbatts that would give "UdevWait waits for any processes that are waiting for udev to complete the specified cookie." ? 

ETOOMUCHWAIT :sweat_smile: